README.md

```diff
@@ -162,6 +162,21 @@ Whether the system is connected to Insights; valid values are `present`
 Whether the system automatically updates the dynamic configuration. It is
 enabled by default.
 
+```yaml
+    rhc_insights:
+      ansible_host: "example-host"
+```
+
+Configures the ansible host name with a custom value for the system record 
+in Host Based Inventory (HBI). This host name is used in playbooks by remediations.
+It is `null` by default and will use the system host name if not specified.
+
+Possible values of this variable:
+
+* `null` or an empty value: the ansible host name is not changed from the system host name.
```

Comment:
I think this could be a bit confusing.
If you have previously set a ansible host name, and run a new playbook with this `null` parameter. I will expect the name to change to the system host name.

```suggestion
* `null` or an empty value: the ansible host name is not changed.
```

Reply:
Yeah this makes sense to me, thanks!